recipes/intxx/bld.bat

```diff
@@ -0,0 +1,31 @@
+
+cmake %CMAKE_ARGS% ^
+  -G "Ninja" ^
+  -S %SRC_DIR% ^
+  -B build ^
+  -D CMAKE_BUILD_TYPE=Release ^
+  -D CMAKE_INSTALL_PREFIX="%PREFIX%" ^
+  -D CMAKE_INSTALL_LIBDIR="Library\lib" ^
+  -D CMAKE_INSTALL_INCLUDEDIR="Library\include" ^
+  -D CMAKE_INSTALL_BINDIR="Library\bin" ^
+  -D CMAKE_INSTALL_DATADIR="Library\share" ^
```

Comment:
```suggestion
  -D CMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
```

Reply:
Ok, done. I usually stick with `%PREFIX%` so the layout is clearer for Library/ and Lib/ with ordinary and Python code, respectively. But you're right, any Python for OpenOrbitalOptimizer will be in a separate bld-py.bat, so LIBRARY_PREFIX is clear here.